src/lib/containers/QueryWrapper.tsx

```diff
@@ -447,6 +467,7 @@ export default class QueryWrapper extends React.Component<
       topLevelControlsState: this.state.topLevelControlsState,
       isColumnSelected: this.state.isColumnSelected,
       selectedRowIndices: this.state.selectedRowIndices,
+      isFacetsAvailable: this.state.isFacetsAvailable,
```

Comment:
isFacetsAvailable state is passed down to QueryWrapper children (for convenience)